databricks_cli/pipelines/cli.py

```diff
@@ -52,40 +52,65 @@
                short_help='Deploys a delta pipeline according to the pipeline specification')
 @click.argument('spec_arg', default=None, required=False)
 @click.option('--spec', default=None, type=PipelineSpecClickType(), help=PipelineSpecClickType.help)
+@click.option('--allow-duplicate-names', is_flag=True,
+              help="Skip duplicate name check while deploying pipeline")
+@click.option('--no-update-spec', is_flag=True,
+              help="Do not update spec with pipeline ID after creating a new pipeline.")
 @debug_option
 @profile_option
 @pipelines_exception_eater
 @provide_api_client
-def deploy_cli(api_client, spec_arg, spec):
+def deploy_cli(api_client, spec_arg, spec, allow_duplicate_names, no_update_spec):
     """
     Deploys a delta pipeline according to the pipeline specification. The pipeline spec is a
     specification that explains how to run a Delta Pipeline on Databricks. All local libraries
     referenced in the spec are uploaded to DBFS.
 
+    The deploy command creates a new pipeline and adds the ID of this pipeline to your spec if your
+    spec does not already contain an ID. The spec will not be updated if the --no-update-spec
+    option is added.
```

Comment:
Minor: To make this a little clearer:
```
If the pipeline spec contains an "id" field, attempts to update an existing pipeline with that ID. If it does not, creates a new pipeline and edits the spec to add the ID of the created pipeline. The spec will not be updated if the --no-update-spec option is added.

The deploy command will not create a new pipeline if a pipeline with the same name already exists. You can disable this check by using the --allow-duplicate-names option.
```

Reply:
Thanks! This is definitely more clear. Updated the comment.